### ISSUE TITLE
A thanks and a release

### ISSUE THREAD

**Issue by myoung34** [myoung34/vagrant-ovirt4#138]
https://github.com/myoung34/vagrant-ovirt4/releases/tag/v2.0.0

If youd all like to test that version and see if all the changes work as expected.

@mika-b 
@dafritz84
@cliffe 
@alclonky 
@GervaisdeM

**Comment by mika-b** [comment]
Thanks for the release.

Some things quickly tested with oVirt 4.3:
 - SSH into existing guest
 - Create new CentOS 7 guest (eth0 configured)
 - Create new Ubuntu bionic guest (ens3 and ens4 configured, rsync working as expected)
 - 'vagrant up' for an existing CentOS 8 guest (rsync working as expected)
 
Used plugin from rubygems (not self built):
vagrant plugin install --plugin-version 2.0.0 vagrant-ovirt4

**Comment by daFritz84** [comment]
Thanks again for the release. I also used the plugin from rubygems.

I tested every command supported with ovirt 4.4.2.6-1.el8:

- `vagrant up` OK
- `vagrant destroy` OK
- `vagrant ssh` OK
- `vagrant ssh-config` OK
- `vagrant halt` OK
- `vagrant reload` OK
- `vagrant status` OK
- `vagrant suspend` OK
- `vagrant resume` OK

The snapshot functionality needs some more love, but there is already an issue for that. 
Unfortunately, I'm back on the job again so I don't have  a lot of time to spare.

**Comment by cliffe** [comment]
Fantastic. Not sure when I'll get a chance to test this (at least a few weeks), but really happy to see some development happening, as we do rely on vagrant-ovirt4 pretty extensively. Thanks for keeping the project alive, @myoung34!

**Comment by myoung34** [comment]
I'm actively trying to find a VPS that does nested virtualization so i can test this myself, but time and commitment is proving hard to make that work

**Comment by daFritz84** [comment]
Uhoh, nested virtualization and ovirt was a f*****g nightmare. I wouldn't go down that road

I did a nested-kvm setup during initial testing of ovirt and I can't remember all kernel parameters I had to set on the VMs to get it finally installed. I had problems from ovirt chocking on the exposed cpu features to gluster not handling xfs xattr in a proper manner.

I would go hyperconverged on a single machine.
